app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt

```diff
@@ -17,131 +17,59 @@
 package co.condorlabs.customcomponents.helper.masks
 
 import android.text.Editable
+import android.text.TextWatcher
 import android.widget.EditText
-import co.condorlabs.customcomponents.*
-import co.condorlabs.customcomponents.helper.TextWatcherAdapter
+import co.condorlabs.customcomponents.EMPTY
+import co.condorlabs.customcomponents.MONEY_MAX_AMOUNT
 import co.condorlabs.customcomponents.helper.equalThan
 import co.condorlabs.customcomponents.helper.lessThan
-import co.condorlabs.customcomponents.helper.toDollarAmount
 import java.math.BigDecimal
-import java.math.RoundingMode
-import java.text.DecimalFormat
 import java.text.NumberFormat
-import java.util.*
 
 /**
  * @author Oscar Gallon on 12/20/18.
  */
-class PriceTextWatcherMask(private val receiver: EditText) : TextWatcherAdapter() {
+class PriceTextWatcherMask(private val receiver: EditText) : TextWatcher {
```

Comment:
verify in the class which constants can be deleted

Reply:
Done